test/infrastructure/docker/api/v1alpha4/dockercluster_types.go

```diff
@@ -42,6 +42,30 @@ type DockerClusterSpec struct {
 	// controllers to do what they will with the defined failure domains.
 	// +optional
 	FailureDomains clusterv1.FailureDomains `json:"failureDomains,omitempty"`
+
+	// LoadBalancer allows defining configurations for the cluster load balancer.
+	// +optional
+	LoadBalancer DockerLoadBalancer `json:"loadBalancer,omitempty"`
+}
+
+// DockerLoadBalancer allows defining configurations for the cluster load balancer.
+type DockerLoadBalancer struct {
+	// ImageMeta allows customizing the image used for image used for the cluster load balancer.
+	ImageMeta `json:",inline"`
+}
+
+// ImageMeta allows customizing the image used for components that are not
+// originated from the Kubernetes/Kubernetes release process.
```

Comment:
This comment seems to be a bit generic compared to the go doc comments for the individual fields.

I wonder if we should call this struct DockerLoadBalancerImageMeta as it's not reusable in other contexts with the current go docs.

Reply:
We have this pattern a few other places:

https://github.com/kubernetes-sigs/cluster-api/blob/master/bootstrap/kubeadm/api/v1alpha4/kubeadm_types.go#L165
https://github.com/kubernetes-sigs/cluster-api/blob/master/bootstrap/kubeadm/types/v1beta2/types.go#L186
https://github.com/kubernetes-sigs/cluster-api/blob/master/bootstrap/kubeadm/types/v1beta1/types.go#L174
https://github.com/kubernetes-sigs/cluster-api/blob/master/bootstrap/kubeadm/types/v1beta3/types.go#L170

Happy to change it, but it would be good to get opinions as to whether we want to lean towards consistency or clarity.

Reply:
Fine for me to keep as is. I think if nobody has a strong opinion as is, let's keep it as is.

Reply:
Great, sounds good Stefan.

If anyone does have a strong opinion, just let me know. Happy to go through the other instances too and rename them to something more specific.